src/Illuminate/Database/Eloquent/Relations/HasManyThrough.php

```diff
@@ -331,7 +331,7 @@ public function firstWhere($column, $operator = null, $value = null, $boolean =
      * Execute the query and get the first related model.
      *
      * @param  array  $columns
-     * @return mixed
+     * @return \Illuminate\Database\Eloquent\Model|static|null
```

Comment:
```suggestion
     * @return ?\Illuminate\Database\Eloquent\Model
```

Reply:
We don't use the `?` in docblocks itself. `null` is what we use.

Reply:
> We don't use the `?` in docblocks itself. `null` is what we use.

Updated!

Reply:
@driesvints can you confirm in `static` can be dropped? Then I can create a follow-up PR for it

Reply:
I'd just leave things be for now. Thanks